CHANGELOG.rst

```diff
@@ -13,9 +13,17 @@ This project adheres to `Semantic Versioning`_ starting with version 0.11.0.
 
 Added
 -----
+- add unit tests of endpoints, actions and utils
+- add support of multiple sanic workers
+- add support of async run method inside the action class
+- add typing for methods across the package
+- add handler of unknown action
+- add handler of empty body requests sent to webhook endpoint
```

Comment:
Please also move this to ``Changed``
```suggestion
- Return status code ``400`` in case an empty request body was sent to the ``/webhook`` endpoint
```